garage/tf/q_functions/mlp_q_function.py

```diff
@@ -0,0 +1,47 @@
+import tensorflow as tf
+
+from garage.misc.overrides import overrides
+from garage.tf.core.networks import mlp
+from garage.tf.q_functions import QFunction
+
+
+class MLPQFunction(QFunction):
```

Comment:
Can we replace ContinuousMLPQFunction with MLPQFunction?

Reply:
What does the merge layer do in ContinuousMLPQFunction?